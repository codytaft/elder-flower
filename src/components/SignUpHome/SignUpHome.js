import React, { Component } from 'react';
import PropTypes from 'prop-types';
import { NavLink } from 'react-router-dom';
import './SignUpHome.css';
export class SignUpHome extends Component {
  render() {
    return (
      <section className="signup-btns">
        <NavLink
          exact
          to="/sign-up-elder"
          className="nav-link nav-link-signup-elder"
        >
          Elder
        </NavLink>
        <NavLink
          exact
          to="/sign-up-carer"
          className="nav-link nav-link-signup-carer"
        >
          Carer
        </NavLink>
      </section>
    );
  }
}
