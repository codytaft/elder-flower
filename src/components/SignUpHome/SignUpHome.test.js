import React from 'react';
import { shallow } from 'enzyme';
import { SignUpHome } from './SignUpHome';

describe('SignUpHome', () => {
  it('should match the snapshot', () => {
    let wrapper;
    wrapper = shallow(<SignUpHome />);
    expect(wrapper).toMatchSnapshot();
  });
});
