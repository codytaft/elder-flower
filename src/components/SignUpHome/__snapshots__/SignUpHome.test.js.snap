// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignUpHome should match the snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <SignUpHome />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <NavLink
          activeClassName="active"
          aria-current="page"
          className="nav-link nav-link-signup-elder"
          exact={true}
          to="/sign-up-elder"
>
          Elder
</NavLink>,
        <NavLink
          activeClassName="active"
          aria-current="page"
          className="nav-link nav-link-signup-carer"
          exact={true}
          to="/sign-up-carer"
>
          Carer
</NavLink>,
      ],
      "className": "signup-btns",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "activeClassName": "active",
          "aria-current": "page",
          "children": "Elder",
          "className": "nav-link nav-link-signup-elder",
          "exact": true,
          "to": "/sign-up-elder",
        },
        "ref": null,
        "rendered": "Elder",
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "activeClassName": "active",
          "aria-current": "page",
          "children": "Carer",
          "className": "nav-link nav-link-signup-carer",
          "exact": true,
          "to": "/sign-up-carer",
        },
        "ref": null,
        "rendered": "Carer",
        "type": [Function],
      },
    ],
    "type": "section",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <NavLink
            activeClassName="active"
            aria-current="page"
            className="nav-link nav-link-signup-elder"
            exact={true}
            to="/sign-up-elder"
>
            Elder
</NavLink>,
          <NavLink
            activeClassName="active"
            aria-current="page"
            className="nav-link nav-link-signup-carer"
            exact={true}
            to="/sign-up-carer"
>
            Carer
</NavLink>,
        ],
        "className": "signup-btns",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "activeClassName": "active",
            "aria-current": "page",
            "children": "Elder",
            "className": "nav-link nav-link-signup-elder",
            "exact": true,
            "to": "/sign-up-elder",
          },
          "ref": null,
          "rendered": "Elder",
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "activeClassName": "active",
            "aria-current": "page",
            "children": "Carer",
            "className": "nav-link nav-link-signup-carer",
            "exact": true,
            "to": "/sign-up-carer",
          },
          "ref": null,
          "rendered": "Carer",
          "type": [Function],
        },
      ],
      "type": "section",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    "attachTo": undefined,
    "hydrateIn": undefined,
  },
}
`;
